nabla-pendulum/deploy/04_deploy_curves.ts

```diff
@@ -0,0 +1,44 @@
+import { WasmDeployEnvironment } from "../../src/index";
+
+const CURVE_PARAMETERS_LOW_SLIPPAGE = {
+  alpha: 0.02,
```

Comment:
In [Notion](https://www.notion.so/satoshipay/24-05-16-Nabla-Pendulum-Deployment-c7cefd58f44c428a92ceaea30c603286?pvs=4#34065ddf8c894229918571c3dda8f1e4) we have `0.001` for each of the low slippage parameters, is this intentionally different?

Reply:
We already have two different Pendulum deployments and will have at least one more. Since this codebase only has one deployment configuration, it can at most represent one of the three deployments. In this sense, it is no historical document of all existing deployments and we need to track that somewhere else (e.g., Notion). Therefore, I don't personally find it important that the current code reflects any of the existing deployments.

Reply:
No problem of course. I just wanted to be sure that it was a different parameter on purpose.

Reply:
Anyway, I changed it back now so that it at least reflects the last deployment, I understand that it is confusing otherwise.